conf/devtest.config

```diff
@@ -0,0 +1,28 @@
+/*
+ * -------------------------------------------------
+ *  nf-core/cageseq Nextflow base config file
+ * -------------------------------------------------
+ * Config for testing on local workstation / laptop
+ */
+
+process {
+
+  cpus = { check_max( 1 * task.attempt, 'cpus' ) }
+  memory = { check_max( 8.GB * task.attempt, 'memory' ) }
+  time = { check_max( 2.h * task.attempt, 'time' ) }
+
+  errorStrategy = { task.exitStatus in [143,137,104,134,139] ? 'retry' : 'finish' }
+  maxRetries = 1
+  maxErrors = '-1'
+
+  // Process-specific resource requirements
+  // See https://www.nextflow.io/docs/latest/config.html#config-process-selectors
+}
+
```

Comment:
Can delete this file because it looks like a custom config that will be entirely dependent on a specific users resources.